Sources/ParseLiveQuery/Internal/QueryEncoder.swift

```diff
@@ -22,15 +22,15 @@ extension Dictionary where Key: ExpressibleByStringLiteral, Value: AnyObject {
         }
         if let conditions: [String:AnyObject] = queryState?.value(forKey: "conditions") as? [String:AnyObject] {
             self["where"] = conditions.encodedQueryDictionary as? Value
-        }
+        } else { self["where"] = [:] as? Value }
```

Comment:
Add a break here so it can follow convention.

} else {
  self["where"] = [:] as? Value
}

Reply:
Sure. I am actually planning on getting rid of this extension anyway. We do 0 checks for even className much less anything else during this process. Even if we did the extension makes it so we cant do Error handling. This entire thing could definitely use some refactoring anyway. Could be a lot more functional.

Reply:
Also we cant add unit tests for just like the .Or query functionality that was added yesterday when the decoding process happens inside an extension statement.

Reply:
Right now I'm just pushing out a way overdue update and I'll start on major changes to get it on swift 5.0 and newer versions of Starscream. This was supposed to go out months ago.

Reply:
Let’s focus on one PR at a time. If you want open a new project with TODO items and we can build a road map.

https://github.com/parse-community/ParseLiveQuery-iOS-OSX/projects

Reply:
Thanks @dplewis! Id totally love a hand.  I'm new to contributing to you guys and public repositories. Been using Parse though for a long time. In fact my very first commit to any public repository was to Parse Live Query when it first came out in 2016. Been using it ever since. In fact thats the reason I'm wanting to do this. I noticed you guys have been stretched pretty thin. I've literally built a career off workin on parse. So I figured I'd try to help you guys out for all the years your guys hard work helped feed my family lol.

This one was practically already done. They had tried to get this released two months ago but couldn't really figure out how to make that happen.

Reply:
I saw your first commit was back in 2016. We are all working towards the same goal. Welcome to the community!